### PR TITLE
purged google analytics

### PR DESCRIPTION
**Pull request by e2002e**
Guten abend !
I have done as you suggested, I'm really satisfied with that, the code was not a pain at all to work with, you have good pupils !
Can you also look at my latest closed issue on your repo ? there is a library that is needed and doesn't compile alongside the rest, we have to go in the folder and manualy make it, i fail to spot what needs to be done here.
Can you also advise on the proper way to get monitor mode with a nexus 7 that has a bcm4330 chipset ? I would enhance the code to make it straightforward for users of this device.
Bis nachste mal !

### CODE REVIEW

File: app/app/src/main/java/de/tu_darmstadt/seemoo/nexmon/gui/ToolsFragment.java [hunk 1]
```diff
@@ -336,69 +290,33 @@ public void run() {
...
-                        tracker.send(new HitBuilders.EventBuilder()
-                                .setCategory("Tools")
-                                .setLabel("Aircrack")
-                                .setAction("install")
-                                .build());
                     }
 
                     if (chkNetcat.isChecked()) {
                         //toast("Installing netcat ...");
-                        copyExtractedAsset(binInstallLocation, "nc");
```
Comment (matthiasseemoo) [inline app/app/src/main/java/de/tu_darmstadt/seemoo/nexmon/gui/ToolsFragment.java:349]: Why did you remove this line?

Comment (e2002e) [reply app/app/src/main/java/de/tu_darmstadt/seemoo/nexmon/gui/ToolsFragment.java:349]: Oh, I went crazy on ctrl+k with nano, many analytics commands in that file.
Removing the last google file.
Got monitor mode active on my device by copying the firmware from bcmon application, no injection support is recognised but i can run the tools (at least on nougat, that's why I'm trying to get the minsdk to 21 or 22 to test on stock lollipop)

**Comment by e2002e** [comment]
Do I have to close the pull request and open a new one with the new commits ?

**Comment by e2002e** [comment]
On my fork I have done three things:
Removed google analytics OK
Set the app to extract the assets in a folder instead of the sdcard's root OK
Tried to get injection mode for the nexus 7 2012: here I would need advise,
everything is setup for it, for i'm not too sure about the firmware, I know this is a bcm4330 but the S2 one is not compatible, also the one from bcm4330 is not, both of them make wifi unusable, I included fw_bcmdhd.bcmon.bin from bcmon and this one is ok, but still no injection.

**Comment by e2002e** [comment]
Hey, read the last commit on master. All should be good.
Now i'm branching on 'nexus 7', I don't see why we couldn't get the injection capabilities for this one.
They had the same workflow: a helper program to get in monitor mode that was called "enable_bcmon" and a library to fake the device called "libfake_driver.so", the source are not available for these I'm afraid so I have to extract the apk file and get them, I just hope nexutil and libfakeioctl can do the work with the firmware or this is lost. CU

**Comment by e2002e** [comment]
Injection is working ! Doesn't need the old firmware from bcmon !
Nexmon doesn't recognise the injection capabilities though. From a terminal I need to call nexutil -m1 and start airodump-ng, in another terminal I can inject deauth requests with aireplay. Of course both commands need libfakeioctl.so to be loaded. Everytime I did that I got the handshake, I was doubtfull so I tried to run airodump without using aireplay and did no get the handshake.
I am going to either work with nexmon's code or make a gui of my own. Depends if you'r willing to support this, in either case I'll obviously use nexutil and libfakeioctl. Have a good evening.

**Comment by matthiasseemoo** [comment]
I do not think that it worked, as our patch for the bcm4330 does not support injection.

**Comment by e2002e** [comment]
I'm using the firmware bcm4330 version 5.90.195.114 it ships with stock rom and also unlegacy android 6 which I have installed.
I think that what I'm going to do is install bcmon and use a linker fix to bypass mandatory pie execution. I hope you take some time to get the injection working with the nexus 7 knowing that nexmon can be installed on it. Aufwierdesehen =)

**Comment by e2002e** [comment]
I just realized that left a sting concerning the nexus 7 in a XML file, there is also a piece of code in a Toast that I  changed to run on lollipop, I will cleanup and reopen pull request.

**Comment by e2002e** [comment]
I think you should merge that or close it, otherwise I will
